Runtime/Interaction/StaticMeshCollidersManager.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using Unity.Jobs;
+using UnityEngine;
+using UnityEngine.Jobs;
+
+namespace ubco.ovilab.HPUI.Interaction
+{
+    /// <summary>
+    /// Component to manage the colliders for a given rectangular custom mesh, based on <see cref="Interaction.DeformableSurfaceCollidersManager"/>
+    /// </summary>
+    public class StaticMeshCollidersManager : MonoBehaviour
+    {
+        [SerializeField] private int[] vertexRemapData;
```

Comment:
should this have a `HideInInspector` attribute?

Reply:
done